app/helpers/tooltips_helper.rb

```diff
@@ -8,15 +8,15 @@ def tooltip_shell(help_text)
     end
   end
 
-  def dashboard_table_content_tooltip_shell(title)
-    return if title == 'Search results'
+  def dashboard_table_content_tooltip_shell(table_type)
```

Comment:
It makes more sense to use the non-localized table_type here, so we're rolling with that.